ray-operator/controllers/ray/common/job.go

```diff
@@ -115,6 +119,10 @@ func GetK8sJobCommand(rayJobInstance *rayv1.RayJob) ([]string, error) {
 	}
 	k8sJobCommand = append(k8sJobCommand, commandSlice...)
 
+	k8sJobCommand = append(k8sJobCommand, "2>&1", "|", "grep", "-zv", "'Please use a different submission_id'")
```

Comment:
Does this obsure any output of the normal case where `ray job submit` doesn't fail due to duplicate submission ID?

Reply:
No, it will only hide the output of duplicate submission. Any other cases will still be printed.

Reply:
For example:
```sh
echo "
normal output 1
normal output 2
normal output 3
" | grep -vz 'Please use a different submission_id'
```

still outputs:

```
normal output 1
normal output 2
normal output 3
```

Reply:
If there's no duplicate submission ID, will it print the logs twice?

Reply:
It won’t because we use --no-wait flag when submitting the job. The flag skips log tailing.

Reply:
Here is an output example of a normal case:

```sh
2024-12-02 18:19:21,800	INFO cli.py:36 -- Job submission server address: http://rayjob-sample-raycluster-bc774-head-svc.default.svc.cluster.local:8265
2024-12-02 18:19:22,405	SUCC cli.py:60 -- ------------------------------------------------
2024-12-02 18:19:22,405	SUCC cli.py:61 -- Job 'rayjob-sample-wvtph' submitted successfully
2024-12-02 18:19:22,405	SUCC cli.py:62 -- ------------------------------------------------
2024-12-02 18:19:22,405	INFO cli.py:285 -- Next steps
2024-12-02 18:19:22,405	INFO cli.py:286 -- Query the logs of the job:
2024-12-02 18:19:22,405	INFO cli.py:288 -- ray job logs rayjob-sample-wvtph
2024-12-02 18:19:22,405	INFO cli.py:290 -- Query the status of the job:
2024-12-02 18:19:22,405	INFO cli.py:292 -- ray job status rayjob-sample-wvtph
2024-12-02 18:19:22,405	INFO cli.py:294 -- Request the job to be stopped:
2024-12-02 18:19:22,406	INFO cli.py:296 -- ray job stop rayjob-sample-wvtph
2024-12-02 18:19:23,626	INFO cli.py:36 -- Job submission server address: http://rayjob-sample-raycluster-bc774-head-svc.default.svc.cluster.local:8265
2024-12-02 18:19:34,567	INFO worker.py:1405 -- Using address 10.244.0.12:6379 set in the environment variable RAY_ADDRESS
2024-12-02 18:19:34,568	INFO worker.py:1540 -- Connecting to existing Ray cluster at address: 10.244.0.12:6379...
2024-12-02 18:19:34,572	INFO worker.py:1715 -- Connected to Ray cluster. View the dashboard at http://10.244.0.12:8265
test_counter got 1
test_counter got 2
test_counter got 3
test_counter got 4
test_counter got 5
2024-12-02 18:19:47,758	SUCC cli.py:60 -- -----------------------------------
2024-12-02 18:19:47,758	SUCC cli.py:61 -- Job 'rayjob-sample-wvtph' succeeded
2024-12-02 18:19:47,758	SUCC cli.py:62 -- -----------------------------------
```